bundle/config/mutator/mutator.go

```diff
@@ -29,7 +28,6 @@ func DefaultMutators(ctx context.Context, b *bundle.Bundle) diag.Diagnostics {
 		ComputeIdToClusterId(),
 		InitializeVariables(),
 		DefineDefaultTarget(),
-		pythonmutator.PythonMutator(pythonmutator.PythonMutatorPhaseLoad),
```

Comment:
nit: If this removal is independent of the rest of the PR you could send it separately to get it merged faster.

Reply:
Not sure if we need to add NEXT_CHANGELOG entry for this?